packages/udes-ui-storybook/storybook/stories/udes-ui-validation/stories.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+// eslint-disable-next-line max-len
+import Material from "../../../../udes-ui-validation/src/exampleComponent/materialUiComponent";
+
+export default [
+  {
+    // eslint-disable-next-line react/display-name
```

Comment:
You can use Storybook knobs here and also move your examples to Storybook pleasde

Reply:
again, strong disagree with moving the entire example to storybook, as it provides the actual example code people may want to reference from the package. Don't really understand the utility of storybook knobs in this instance either but can try adding them.

Reply:
See comment above :)